### PR TITLE
USDC Deployment

### PR DESCRIPTION
**Pull request by smmackey**

### CODE REVIEW

File: addresses/frontend-mainnet.json [hunk 1]
```diff
@@ -26,6 +27,9 @@
         "0xF94A7Df264A2ec8bCEef2cFE54d7cA3f6C6DFC7a",
         "0xb1cc77e701de60FE246607565CF7eDC9D9b6b963",
         "0x71628c66C502F988Fbb9e17081F2bD14e361FAF4",
-        "0xE54B3F5c444a801e61BECDCa93e74CdC1C4C1F90"
+        "0xE54B3F5c444a801e61BECDCa93e74CdC1C4C1F90",
+        "0xf38c3E836Be9cD35072055Ff6a9Ba570e0B70797",
+        "0x787546Bf2c05e3e19e2b6BDE57A203da7f682efF",
+        "0x2d6e3515c8b47192ca3913770fa741d3c4dac354"
```
Comment (DannyDelott) [inline addresses/frontend-mainnet.json:33]: Minor capitalization nit: `0x2D6e3515C8b47192Ca3913770fa741d3C4Dac354`

**Review by DannyDelott — CHANGES_REQUESTED** [review]

File: addresses/frontend-mainnet.json (hunk 1 above)
Comment (jrhea) [reply addresses/frontend-mainnet.json:33]: 😱

File: addresses/mainnet.json [hunk 2]
```diff
@@ -126,6 +128,26 @@
...
+				"address": "0xf38c3E836Be9cD35072055Ff6a9Ba570e0B70797",
+				"trancheFactory": "0x62F161BF3692E4015BefB05A03a94A40f520d1c0",
+				"ptPool": {
+					"address": "0x787546Bf2c05e3e19e2b6BDE57A203da7f682efF",
+					"poolId": "0x787546bf2c05e3e19e2b6bde57a203da7f682eff00020000000000000000007c",
+					"fee": "0.1",
+					"timeStretch": 18.5
+				},
+				"ytPool": {
+					"address": "0x2d6e3515c8b47192ca3913770fa741d3c4dac354",
```
Comment (jrhea) [inline addresses/mainnet.json:144]:
```suggestion
					"address": "0x2D6e3515C8b47192Ca3913770fa741d3C4Dac354",
```

Capitalization nit

File: addresses/frontend-mainnet.json (hunk 1 above)
Comment (jrhea) [inline addresses/frontend-mainnet.json:33]:
```suggestion
        "0x2D6e3515C8b47192Ca3913770fa741d3C4Dac354"
```

**Review by DannyDelott — APPROVED** [review]

**Review by jrhea — APPROVED** [review]

**Pull request merged**